src/ctapipe/calib/camera/calibrator.py

```diff
@@ -61,6 +62,11 @@ class CameraCalibrator(TelescopeComponent):
 
     image_extractor_type: str
         The name of the ImageExtractor subclass to be used for image extraction
+
+    image_calibration_type: str
```

Comment:
I don't think we need this. Instead, you can check the type of the image extractor. 
```
if isinstance(VarianceExtractor, extractor):
    #apply squared gain calibration

Reply:
OK, I'll do that